charts/sda-svc/values.yaml

```diff
@@ -232,6 +232,7 @@ global:
       iss:
         - iss: "https://login.elixir-czech.org/oidc"
           jku: "https://login.elixir-czech.org/oidc/jwk"
+    serveUnencryptedData: false
```

Comment:
This option needs to be documented in the readme file as well.

Reply:
Thanks for pointing this out. Addressed in the commit c6bf828e